stdlib/ast.pyi

```diff
@@ -1721,6 +1721,15 @@ if sys.version_info < (3, 14):
 _T = _TypeVar("_T", bound=AST)
 
 if sys.version_info >= (3, 13):
+    @overload
+    def parse(
```

Comment:
This seems redundant with the next overload. What calls would match this overload but not the next one?